plugins/volumeFilter_test.go

```diff
@@ -155,130 +155,130 @@ func TestMakeFilterVolume(t *testing.T) {
 
 func TestVolumeFilterFn(t *testing.T) {
 	testCases := []struct {
-		name               string
-		mode               volumeFilterMode
-		sellBaseCapInBase  *float64
-		sellBaseCapInQuote *float64
-		otbBase            *float64
-		otbQuote           *float64
-		tbbBase            *float64
-		tbbQuote           *float64
-		inputOp            *txnbuild.ManageSellOffer
-		wantOp             *txnbuild.ManageSellOffer
-		wantTbbBase        *float64
-		wantTbbQuote       *float64
+		name           string
+		mode           volumeFilterMode
+		baseCapInBase  *float64
```

Comment:
PR submission tip:
usually a good idea to do renames in a completely separate PR, even if it's only a 1-line PR. such PRs should include renames of only one type so it's a 1-second review vs something longer, and so it doesn't get held up with other changes such as the one above.

example: `sellBaseCapInBase` -> `baseCapInBase` + `sellBaseCapInQuote` -> `baseCapInQuote` can be in 1 PR, however a hypothetical rename of `sellBaseCapInBase` -> `sellBaseCapInTop` + `sellBaseCapInQuote` -> `sellBaseCapInBottom` should be in two separate PRs to make the review process much much simpler.

does that make sense?

if you prefer extracting out the renames into a separate PR that's ok with me, or if you want to leave here.

Reply:
Yup, this makes a lot of sense. As discussed, I'm changing this PR to just the renames, and I'll roll the buy-side implementation into a separate PR.